etw/rust/tracelogging/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tracelogging"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Since this isn't a breaking API change, shouldn't this be 0.1.1, so that everybody using the crate version "0.1" will still pick up the change when they run cargo update?

Reply:
> The string "0.1.12" is a version requirement. Although it looks like a specific version of the time crate, it actually specifies a range of versions and allows [SemVer](https://semver.org/) compatible updates. An update is allowed if the new version number does not modify the left-most non-zero digit in the major, minor, patch grouping. In this case, if we ran cargo update -p time, cargo should update us to version 0.1.13 if it is the latest 0.1.z release, but would not update us to 0.2.0. If instead we had specified the version string as 1.0, cargo should update to 1.1 if it is the latest 1.y release, but not 2.0. The version 0.0.x is not considered compatible with any other version.

Reply:
Version 1.0.1 seems reasonable at this point.